experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/AmActor.scala

```diff
@@ -86,12 +84,25 @@ class AmActor(appConfig: AppConfig, yarnConf: YarnConfiguration) extends Actor {
     case containerStarted: ContainerStarted =>
       LOG.info(s"Started container : ${containerStarted.containerId}") 
       if(needMoreMasterContainersState) {
-        masterContainersStarted += 1        
+        masterContainersStarted += 1
         LOG.info(s"Currently master containers started : $masterContainersStarted/${appConfig.getEnv(GEARPUMPMASTER_CONTAINERS).toInt}")
-        requestWorkerContainersIfNeeded
+        requestWorkerContainersIfNeeded()
       } else {
         workerContainersStarted += 1
         LOG.info(s"Currently worker containers started : $workerContainersStarted/${appConfig.getEnv(WORKER_CONTAINERS).toInt}")
+        workerContainersStarted == workerContainersRequested match {
```

Comment:
Maybe this simplify this block.

Looks like it contain more lines than expected.